src/components/field/index.stories.tsx

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
 import { Meta, StoryObj } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
-import { Box, Input } from '@chakra-ui/react';
+import { Box, Textarea } from '@chakra-ui/react';
```

Comment:
Why not import `Textarea` from `src`?

Reply:
thx, messed up switching from my temp version